src/mare/compiler/binary.cr

```diff
@@ -24,7 +24,14 @@ class Mare::Compiler::Binary
     machine = ctx.code_gen.target_machine
     mod = ctx.code_gen.mod
 
-    ponyrt_bc = LLVM::MemoryBuffer.from_file(PONYRT_BC_PATH)
+    target = ctx.options.target
+    ponyrt_bc_path = if target && target.freebsd?
+                       "/usr/local/lib/libponyrt.bc"
+                     else
+                       PONYRT_BC_PATH
+                     end
```

Comment:
Instead of switching paths between `/usr/lib` and `/usr/local/lib` based on the target, maybe we should just have a list of paths we look in, and we take whichever one we find a `libponyrt.bc` in first?

In the future when we add cross-compiling we'll probably need to have to bundle in our distro package a precompiled runtime bitcode for every platform we support, and just choose the correct one here based on what target was selected.

But for now I suggest just having multiple search paths and not using `target` to influence the bitcode that gets selected.